### PR TITLE
Hawq 1396. Fix the bug when query hcatalog via PXF, add test

### PR DESCRIPTION
**Pull request by interma**
add hcatalog test

### CODE REVIEW

**Comment by interma** [comment]
@linwen @ictmalili @wcl14 @stanlyxiang help to review, thanks.

File: src/test/feature/Ranger/test_ranger.cpp [hunk 1]
```diff
@@ -314,6 +314,35 @@ TEST_F(TestHawqRanger, ResourceIncludeATest) {
...
+		Command::getCommandStatus(cmd);
+
+		/*
+		 * create a user and query this table, fail.
+		 */
+		addUser(&util, "pxf", 1, false);
+		runSQLFile(&util, "pxf", "fail", 1);
+
+		/*
+		 * add an allow policy for this user and query again, succeed.
```
Comment (linwen) [inline src/test/feature/Ranger/test_ranger.cpp]: This line of comment is not accurate. Should be "allow policies"

File: src/test/feature/Ranger/test_ranger.cpp [hunk 2]
```diff
@@ -314,6 +314,35 @@ TEST_F(TestHawqRanger, ResourceIncludeATest) {
...
 }
 
+TEST_F(TestHawqRanger, HcatalogTest) {
+	SQLUtility util;
+	if (util.getGUCValue("hawq_acl_type") == "ranger")
+	{
+		/*
+		 * create a table in hive and populate some rows
+		 */
+		clearEnv(&util, "pxf", 1);
```
Comment (linwen) [inline src/test/feature/Ranger/test_ranger.cpp]: should clean env for 2 and 3, added from line:340

**Comment by interma** [comment]
All fixed. 
Thanks!

**Comment by linwen** [comment]
+1

**Comment by interma** [comment]
merged

**Pull request closed without merging**